pages/_app.js

```diff
@@ -50,6 +63,12 @@ export default class MyApp extends App {
     const { favTeam } = this.state;
     const { games, router } = this.props;
 
+    if (favTeam.length === 0) {
+      router.push('/teams');
+    }
+
+    console.log();
```

Comment:
can we also drop this?

Reply:
nvm just saw these get removed in the next commit. can probably squash the next commit into this one